providers/frame/zombienet.ts

```diff
@@ -82,27 +101,12 @@ export class ZombienetProvider extends FrameProxyProvider {
       })
       const out = new Buffer()
```

Comment:
Perhaps this would work instead:
```ts
const config = await configPending.catch(() => undefined)
if(config) return config
await copy(process.stderr, Deno.stderr)
throw new Error("Zombienet exited without launching network")
```

Reply:
Used `copy` to pipe buffer into `Deno.stderr`, but the other changes result in errors hanging (imports never load).

Reply:
Your way is working